templates/server.properties.erb

```diff
@@ -2,8 +2,10 @@
 
 broker.id=<%= scope.lookupvar('kafka::server::broker_id') %>
 log.dirs=<%= Array(scope.lookupvar('kafka::server::log_dirs')).join(',') %>
-zookeeper.connect=<%= scope.lookupvar('kafka::server::zookeeper_connect') %>
+zookeeper.connect=<%= scope.lookupvar('kafka::server::zookeeper_connect') -%>
+<%- if scope.lookupvar('kafka::server::is_broker_rack_aware') -%>
```

Comment:
We don't need an additional variable. We can use broker_rack to perform this check.
`<% if scope.lookupvar('kafka::server::broker_rack') %>broker.rack=<%=scope.lookupvar('kafka::server::broker_rack'%><% end %>`